checkov/common/runners/runner_registry.py

```diff
@@ -210,6 +212,13 @@ def _merge_reports(self, reports: Iterable[Report | list[Report]]) -> list[Repor
                     self._check_type_to_report_map[sub_report.check_type] = sub_report
                     merged_reports.append(sub_report)
 
+                if sub_report.check_type == 'sca_image' and bc_integration.customer_run_config_response:
+                    if len(sub_reports) == 1 or (len(sub_reports) == 2 and sub_reports[0].check_type in bc_integration.customer_run_config_response.get('supportedIrFw', [])):
```

Comment:
this check is really unclear (specifically the second part) - could you separate it to a function with an explanatory name?